mitmproxy/contentviews/image/image_parser.py

```diff
@@ -0,0 +1,34 @@
+from kaitaistruct import KaitaiStream
+
+from . import png
+
+def get_png(data):
```

Comment:
Can you please add type annotations? Also this function needs a better name, e.g. `extract_metadata_from_png`, `parse_png` or `handle_png`, or .... It does not "get" a png. 😃 


```python
Metadata = typing.List[typing.Tuple[str, str]]

def parse_png(data: bytes) -> Metadata:
```